docs/directory.json

```diff
@@ -131,10 +127,6 @@
         {
           "title": "部署 EMQX 集群",
           "path": "getting-started/getting-started"
-        },
-        {
-          "title": "部署 EMQX 插件",
-          "path": "getting-started/install-emqx-prometheus-plugin"
```

Comment:
Do we need delete Markdown file ？

Reply:
> Do we need delete Markdown file ？

I think this article is useless because we have a separate article about prometheus monitoring

Reply:
So we can delete that file